src/core/push/w3c.ts

```diff
@@ -59,7 +59,7 @@ export default class W3cPushManager implements PushOpsManager {
         const workerReg = await getActiveServiceWorkerReg();
         const existingSub = await workerReg.pushManager.getSubscription();
 
-        if (existingSub && !hasSameKey(ctx.vapidPublicKey, existingSub)) {
+        if (existingSub && !hasSameKey(ctx.vapidPublicKey!, existingSub)) {
```

Comment:
Assume related to making the field optional, but if we can avoid force unwraps I'm all for it.

Reply:
Looking from the pov of the consumer of push init, this would seem like a critical failure if not present - I've thrown if this not present which would seem like the correct thing to do based in this value having to be present.  This would mean a consumer calling the SDK programatically via e.g `Kumulos('pushRegister');` would have to handle if this was the case.

Reply:
Well yes if it's not present it's a problem, but what I'm getting at is I think we should never execute this deep into the code _without_ that being present?

Reply:
Must have been thinking about something else here - have reverted the optional modifier due to the init paths that were taken outside doing the spike for this - no longer requires unwrapping 👍